docs/source/changes.rst

```diff
@@ -5,6 +5,47 @@
 Changes
 =======
 
+v0.5.0
+======
+
+**Breaking change**
+
+* Remove the ``BoxCox`` transform. ( `#222`_ )
+
+**New features**
+
+* Add new contribution ``iSMD`` by `qi`_. ( `#208`_ )
```

Comment:
Shouldn't this be ``ISMD``? (all capitalized)
Also, how about making it more specific:
* Add new contribution ``ISMD`` by `qi`_, a new class of ``BaseProposal`` that allows generation of molecules based on virtual reaction of reactants in a predefined reactant pool. ( `#208`_ )

Reply:
@qi-zh  I saw Qi use iSMD somewhere. Let's ask Qi for the answer.

Reply:
If we keep it the same as IQSPR, like you make iQSPR => IQSPR now. So, I would say keep it that way too. I remember that you said we should use the convention wherever possible.